# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/gamepad/ipc/PGamepadEventChannel.ipdl
# Commit: 8132dd939023
# Full Hash: 8132dd9390233328c8da7072f36859424a2b079d
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-10-14 21:42:48
# Regressor Bug: 1657404
# File Overlap Count: 2
# Description:
#   Bug 1657404 - Change PGamepadEventChannel to "refcounted protocol" r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93119
# ==============================================================================

diff -r 186ca9e93e1d -r 8132dd939023 dom/gamepad/ipc/PGamepadEventChannel.ipdl
--- a/dom/gamepad/ipc/PGamepadEventChannel.ipdl	Wed Oct 07 11:34:02 2020 +0000
+++ b/dom/gamepad/ipc/PGamepadEventChannel.ipdl	Wed Oct 14 15:58:36 2020 +0000
@@ -7,7 +7,7 @@
 namespace mozilla {
 namespace dom {
 
-async protocol PGamepadEventChannel {
+refcounted protocol PGamepadEventChannel {
   manager PBackground;
   parent:
     async __delete__();